clientd3d/ownerdrw.c

```diff
@@ -458,7 +458,15 @@ void DrawOwnerListItem(const DRAWITEMSTRUCT *lpdis, Bool selected, Bool combo)
    if (style & OD_COLORTEXT)
    {
      // get the color we'd prefer for this particular obj
-     crColorText = GetPlayerNameColor(obj->flags,NULL);
+     if (obj == NULL) 
+     {
+        crColorText = PALETTERGB(255, 255, 255);
```

Comment:
Just double-checking that this is what you want.  In the PR commentary, you talk about using default black text, but this is white.

Reply:
Yes I think it should be white because in this case we are talking about player colored text or a catchall vs the rarity colors.  I got confused, I think it makes sense to stick with white because a few lines below it talks about having a black halo around the text to make it more visible, so black on black wouldn't work.